plugins/processors/enum/enum.go

```diff
@@ -34,22 +36,24 @@ func (*EnumMapper) SampleConfig() string {
 }
 
 func (mapper *EnumMapper) Init() error {
-	mapper.FieldFilters = make(map[string]filter.Filter)
-	mapper.TagFilters = make(map[string]filter.Filter)
 	for _, mapping := range mapper.Mappings {
+		// Handle deprecated field option
 		if mapping.Field != "" {
-			fieldFilter, err := filter.NewIncludeExcludeFilter([]string{mapping.Field}, nil)
+			mapping.Fields = append(mapping.Fields, mapping.Field)
+		}
+		if len(mapping.Fields) != 0 {
```

Comment:
The length check is not required. `filter.Compile` will return `nil, nil` in case of an empty list...